lib/membrane/video_compositor/scene.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Membrane.VideoCompositor.Scene do
+  @moduledoc """
+  Structure representing a top level specification of what is Video Compositor
+  supposed to render.
+
+  The main part of the Scene are `Membrane.VideoCompositor.Scene.Object`s
+  and interactions between them. There are three kinds of Objects:
+  - `Membrane.VideoCompositor.Scene.Object.Texture` - single input object,
+  taking frames and applying a series of transformations onto it.
+  - `Membrane.VideoCompositor.Scene.Object.Layout` - combining
+  frames from multiple inputs into a single output.
+  - `Membrane.VideoCompositor.Scene.Object.InputVideo` - which maps
+  an input pad of element into Scene object.
+  """
```

Comment:
NIT - this order better follows the usual definition of objects in the scene.
```suggestion
  The main part of the Scene are `Membrane.VideoCompositor.Scene.Object`s
  and interactions between them. There are three kinds of Objects:
   - `Membrane.VideoCompositor.Scene.Object.InputVideo` - which maps
  an input pad of element into Scene object.
  - `Membrane.VideoCompositor.Scene.Object.Texture` - single input object,
  taking frames and applying a series of transformations onto it.
  - `Membrane.VideoCompositor.Scene.Object.Layout` - combining
  frames from multiple inputs into a single output.
  """
```

Reply:
Moreover, I think the second part should rather be in `@type t` `@typedoc`